routes/truckRoutes.js

```diff
@@ -1,3 +1,65 @@
-// connect to mongodb in this module as this is where you'll be making creat/read/delete calls to your database
-// use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
-// remember this is a Node module
+
+var express = require('express');
+var router = express.Router();
+
+var mongoose = require('mongoose');
+var db = mongoose.connect('mongodb://localhost/foodTruckAPI');
+var Truck = require("../models/truckModel");
+
+router.route('/')
+	.get( function (request, response) {
+		Truck.find(function (error, trucks) {
+		if (error) {
+			response.status(500).send(error);
+		} else {
+			response.json(trucks);
+		}
+		});
+	})
+
+	.post(function(request, response){
+		    // the request.body is the new truck object
+	    var truck = new Truck(request.body);
+	    console.log(truck);

```

Comment:
Remember to remove console.log calls before code goes to production. Although it's not as big of a deal as it used to be, old browsers such as IE7 don't have a console when the developer tools aren't open. The console.log would cause an error and potentially break other scripts.